### PR TITLE
[PLA-1342] Adds signingPayload to fields

### PR DESCRIPTION
**Pull request by leonardocustodio**

### CODE REVIEW

File: src/Support/helpers.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?php
+
```
Comment (enjinabner) [inline src/Support/helpers.php:2]: Do we need this helper class? I think we can just put them in Support/Util.php class

**Review by enjinabner — CHANGES_REQUESTED** [review]

File: src/Support/helpers.php (hunk 1 above)
Comment (leonardocustodio) [reply src/Support/helpers.php:2]: I feel this is a bit more clean (shorter) as we don't need the namespace. There are lots of those logic in the code, I did not replace them all but I think this way would look better.

**Review by enjinabner — APPROVED** [review]

File: schema/schema.gql [hunk 2]
```diff
@@ -511,14 +511,21 @@ type Mutation {
...
     """enjin-platform::args.signingAccount"""
     signingAccount: String
 
+=======
+>>>>>>> Adds signingPayload
     """Simulates a transaction without broadcasting it to the network."""
     simulate: Boolean = false
 
     """Skip all validation rules, use with caution. Defaults to false."""
+<<<<<<< master
```
Comment (pawell67) [inline schema/schema.gql:524]: Seems like conflicts were not resolved correctly

Comment (leonardocustodio) [reply schema/schema.gql:524]: This file is not used at all anyway thank you. I will open another PR to fix anyway.

**Pull request merged**